src/prerender.ts

```diff
@@ -112,7 +112,11 @@ export async function prerender(nitro: Nitro) {
 
     // Check for explicitly ignored routes
     for (const ignore of nitro.options.prerender.ignore) {
-      if (route.startsWith(ignore)) {
+      if (typeof ignore === "string") {
+        if (route.startsWith(ignore)) {
+          return false;
+        }
+      } else if (ignore.test(route)) {
```

Comment:
Shall we test if `ignore.test` is a function to make sure it does not break anything?

Reply:
For now, ignore would be neither string or Regex if the provided configuration is checked by TypeScript. So there's no need to test wether `ignore.test` exists.